projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js

```diff
@@ -125,8 +126,9 @@ function NewsletterNotice( {
 			<Notice status="info" isDismissible={ false } className="edit-post-post-visibility__notice">
 				{ createInterpolateElement(
 					sprintf(
-						/* translators: %s is the number of subscribers in numerical format */
-						__( 'This will be sent to <strong>%s subscribers</strong>.', 'jetpack' ),
+						/* translators: 1 Changes the phrase to be future tense or past tense, 2 Number of subscribers */
+						__( 'This post %1$s sent to <strong>%2$s subscribers</strong>.', 'jetpack' ),
```

Comment:
Please add a separate case for post. The way you implemented it will be hard or even impossible for translators to do their job properly. 
```suggestion
if ( isPostPublishPanel ) {
/* translators: %s is the number of subscribers in numerical format */
						__( 'This was sent to <strong>%s subscribers</strong>.', 'jetpack' ),
} else {
/* translators: %s is the number of subscribers in numerical format */
						__( 'This will be sent to <strong>%s subscribers</strong>.', 'jetpack' ),
}


						
```

Reply:
Fixed in [f775fc1](https://github.com/Automattic/jetpack/pull/30668/commits/f775fc165e98358b6fac0016294288d18f546b02)